src/mame/drivers/klax.cpp

```diff
@@ -64,6 +82,13 @@ void klax_state::machine_reset()
 {
 }
 
+MACHINE_START_MEMBER(klax_bootleg_state, klax5bl)
+{
+	m_rombank->configure_entries(0, 4, memregion("audiocpu")->base(), 0x8000);
+
+	m_audio_sample[0] = m_audio_sample[1] = 0x00;
+	m_audio_nibble = 0;
+}
```

Comment:
We’re trying to get away from this pattern.  Please override `machine_start` to provide specific startup behaviour.